CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ include_directories(
     ${CMAKE_CURRENT_BINARY_DIR}/include
     ${CMAKE_CURRENT_BINARY_DIR}/core
     SYSTEM ${additional_includes}
+    SYSTEM libs/json11
```

Comment:
When #188 gets merged, I guess json11 can become a dependency of the mission plugin only, right?

Reply:
Yes, right.